AzureCalling/Views/ParticipantView.swift

```diff
@@ -37,6 +42,12 @@ class ParticipantView: UIView {
         initSubviews()
     }
 
+    // MARK: Actions
+
+    @IBAction func switchCamera(_ sender: UIButton) {
```

Comment:
Since switch camera functionality is only available for local participants, we should consider a separate class for `LocalParticipantView` that extends `ParticipantView`.

Reply:
Good point! That should abstract away some of the logic (including video toggling) pertaining to local participant activities.